py4syn/utils/scan.py

```diff
@@ -897,11 +897,14 @@ def start(self):
                 FILE_WRITER.insertComment(SCAN_COMMENT)
 
                 try:
-                    user = os.getlogin()
-                except FileNotFoundError:
+                    print('getting user from env')
```

Comment:
Are these prints necessary? It looks like debug messages

Reply:
> Are these prints necessary? It looks like debug messages

My bad, I forgot to delete these prints. I am pulling a review branch. :) Thanks!

Reply:
```suggestion
                    print('getting user from env')
                    
```